torch/csrc/jit/codegen/cuda/scheduler/registry.h

```diff
@@ -189,13 +189,13 @@ class TORCH_CUDA_CU_API SchedulerEntry {
   const bool has_reduction_param_;
 
   //! Reduction parameters if applicable
-  ReductionParams rparams_;
+  ReductionParams rparams_ = ReductionParams();
```

Comment:
nit: isn't the constructor called by default?

Reply:
I think so. maybe there's something with linter at the time of PR.
Let's ditch the change then. Hopefully linter would be quieter next time. :crossed_fingers: